src/test.ts

```diff
@@ -0,0 +1,10 @@
+// We are shifting over to namespaced exports
+export * as testUtilsForum from './lib/testing/forums';
+export * as testUtilsMembers from './lib/testing/members';
+export * as testUtilsPages from './lib/testing/pages';
+export * as testUtilsProposals from './lib/testing/proposals';
+export * as testUtilsRandom from './lib/testing/random';
+export * as testUtilsSpaces from './lib/testing/spaces';
+export * as testUtilsUser from './lib/testing/user';
+export * as testUtilsBounties from './lib/testing/bounties';
+export * as testUtilsVotes from './lib/testing/votes';
```

Comment:
In this example (contrary to permissions.ts) I explicitly selected the exports directly from the top level file, rather than having a local index.ts